framework/tlvf/AutoGenerated/include/tlvf/test/tlvVarList.h

```diff
@@ -33,10 +33,10 @@ class tlvTestVarList : public BaseClass
         const uint8_t& type();
         const uint16_t& length();
         uint16_t& var0();
-        uint16_t& simple_list_length();
```

Comment:
Why 2 autogenerated commits one after the other? looks strange, the previous one did not change tlvf.py. Am I missing something?

Reply:
That's because github doesn't display commits in the correct order. They are not in the actual order, but instead they are ordered by date. For this particular commit, I accidentally copied the timestamp of the previous autogenerated commit (by using `-c HEAD^` because I was too lazy to type the same commit message again). So the commit date of this commit is before the commit date of the change to tlvVarList that triggers it.

Anyway, I'm going to rebase and squash, like you proposed, so the problem will go away :-)